recipes/scikit-plots/meta.yaml

```diff
@@ -0,0 +1,163 @@
+{% set name = "scikit-plots" %}
+{% set version = "0.4.0rc4" %}
+{% set tag = "v" ~ version %}  # sometimes used to match PyPI releases
+
+# Update the numpy value here to the most recent released version
+# when updating to a new scipy feature release; override default pins
+{% set numpy = "2.2.1" %}
+{% set numpy_minor = numpy.split(".")[1] | int %}
+
+# ensure is_freethreading gets detected as a used variable
+# [is_freethreading]
+
+{% if CI is not defined %}{% set CI=False %}{% endif %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:  # Need one of source dist (github, pypi, pypi.anaconda)
+  ## https://github.com/{{ name }}/{{ name }}/archive/{{ tag }}.tar.gz  # ❌ not include submodules
+  ## https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz  # ✅ include submodules
+  ## https://pypi.anaconda.org/scikit-plots-wheels-staging-nightly/simple/{{ name }}/{{ version }}/{{ name | replace("-", "_") }}-{{ version }}.tar.gz  # ✅ include submodules
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive:
+  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+  # url: https://github.com/{{ name }}/{{ name }}/archive/{{ tag }}.tar.gz
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+  # sha256: 6e3731d677b25f7e1a3529d591906effaaf31f4392d66280226f1515ba8f1e10
+  # Conda Recipe: Specify the tag in the git_url using the @tag syntax.
+  git_url: https://github.com/scikit-plots/scikit-plots.git
+  git_rev: {{ tag }}       # 🔒 use a tag or commit hash for reproducibility
+  #  fetching only the latest commit (or the commit specified by git_rev) and no history
+  # git_depth: 1             # (optional) shallow clone for faster builds to speed up CI
```

Comment:
Please clear out all of these comments from the example recipe if they are irrelevant to your package.